### ISSUE TITLE
Issues running the cmdlet

### ISSUE THREAD

**Issue by markkinsella** [MicrosoftDocs/dynamics365smb-devitpro-pb#1815]
Hi, can you advise if there is any special permissions/setup required (on the database or app server) for running this cmdlet? I'm having problems getting it to run at all (step 4). Error attached.

I've confirmed the paths in the command (client path, output path) are correct.
I have local admin access on the machine i'm running the script.
PowerShell is run as administrator.
I've confirmed the database server and database name are correct.
I've moved the client folder to a new location to reduce the length of the path name just in case.
I've confirmed that my DEV client and license allow me to export objects to txt file without issue.

![NAVUATCANSERV01 - Remote Desktop Connection Manager v2 7](https://user-images.githubusercontent.com/83023248/115737255-df759380-a383-11eb-9f5f-cdcd17ab9c03.jpg)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 00ecb53f-3816-6c06-7685-97060cec6cff
* Version Independent ID: b30de908-8377-e4e6-96e4-eb1c5a7ed9b9
* Content: [Making Pages and Reports in the MenuSuite Searchable After Upgrade - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/upgrade/upgrade-pages-report-for-search)
* Content Source: [dev-itpro/upgrade/upgrade-pages-report-for-search.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/main/dev-itpro/upgrade/upgrade-pages-report-for-search.md)
* Service: **dynamics365-business-central**
* GitHub Login: @SusanneWindfeldPedersen
* Microsoft Alias: **solsen**

**Comment by jswymer** [comment]
Hello,

The permissions and setup look correct. Based on the errors, the only thing I can think of is that it has to do with the RoleTailoredClientFolder. The script uses the finsql.exe, which is the Developer Environment (CSIDE). Does the RTC folder include the finsql.exe?

**Comment by markkinsella** [comment]
hi @jswymer , thanks for the reply.

Yes the DEV client executable finsql.exe is in that folder, I've run that client directly from that folder myself to make sure it works and it does, i've also tried manually exporting objects to txt file and it works fine also. The version of BC is 14.0.37587 which is v14 CU9 I believe.

Thanks.

**Comment by jswymer** [comment]
Hello,

It may have to do with User Access Control settings, but not sure..

**Comment by jswymer** [comment]
Hello, 

I just wanted to follow up on this issue, and hear whether you had any success by changing the User Access Control settings on your machine.

**Comment by markkinsella** [comment]
Hi @jswymer 

Thanks for following up. Changing the UAC settings didn't make any difference I'm afraid. In the end I had to use a work around of sorts. 

I took a backup of the BC database, restored it and cleared it down of data. I then installed a BC environment on my laptop and restored this DB into SQL express on my laptop. I was then able to run the cmdlet and update my Page/Report objects. I then just exported the objects from this local DB and imported them back in to my main DB. 

There must be something here though that should be called out that prevents the cmdlet from running on a Server. I tried two different Windows Servers and the cmdlet would not run on either. I had full admin access to both servers. Both servers were BC service tier servers.

Thanks again.
